src/components/molecules/PollBox/PollBox.tsx

```diff
@@ -36,7 +38,7 @@ export const PollBox: React.FC<PollBoxProps> = ({ createPoll }) => {
       topic,
       questions: questions.map(({ name }, id) => ({ name, id })),
     });
-    reset();
+    unselectOption();
   });
```

Comment:
Instead of passing in both `createPoll` and `unselectOption` to this component to then create `onPollSubmit` here, we should just be passing in a single prop that already combines them as a single handler.

This is an 'implementation detail' of the calling component (`src/components/molecules/Chatbox/Chatbox.tsx`) that is 'leaking' down into this component currently. This component shouldn't need to 'know/think about/care about' it. It should just know that it needs to call a single handler function when the form is submitted.